creation/create_cvmfsexec_distros.sh

```diff
@@ -3,123 +3,196 @@
 # SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
 
+# This script generates cvmfsexec distributions for various cvmfs configurations
+# and supported machine types, as supported by the open-source cvmfsexec utility.
+
+# Hardcoded variables
+CVMFSEXEC_REPO="https://www.github.com/cvmfs/cvmfsexec.git"
+DEFAULT_WORK_DIR="/var/lib/gwms-factory/work-dir"
+# TODO: add rhel9, other rhel derivatives and opensuse derivatives to the default list below eventually
+DEFAULT_MACHINE_TYPES="rhel7-x86_64,rhel8-x86_64,suse15-x86_64"
+
 usage() {
-	echo "This script is used to generate cvmfsexec distributions for all"
-	echo "supported machine types (platform- and architecture-based)."
-	echo "The script takes one parameter {osg|egi|default} which specifies"
-	echo "the source to download the latest cvmfs configuration and repositories."
+	echo -e "\nUsage: $0 [--work-dir DIR] SOURCES_LIST [PLATFORMS_LIST]"
+	echo ""
+	echo "DIR: full, absolute path to the factory work directory"
+	echo ""
+	echo "SOURCES_LIST (required): specifies the source(s) to download the latest"
+	echo "cvmfs configuration and repositories from. Must be at least one value "
+	echo "or a comma-separated list of values from the options {osg|egi|default}."
+	echo ""
+	echo "PLATFORMS_LIST (optional): indicates machine types (platform- and architecture-based)"
+	echo "for which distributions is to be built. Can be empty, a single value or a "
+	echo "comma-separated list of values from the options {rhel7-x86_64|rhel8-x86_64|suse15-x86_64}."
```

Comment:
You may consider heredocs when printing many lines. 
```
cat << EOF
Usage: $0 [--work-dir DIR] SOURCES_LIST [PLATFORMS_LIST]
...
EOF
```

Reply:
updated, thanks